client/src/main/java/com/battleships/gamewindow/board/PlayerBoard.java

```diff
@@ -0,0 +1,43 @@
+package com.battleships.gamewindow.board;
+
+import com.battleships.gamewindow.board.fieldStates.BoardField;
+import com.battleships.gamewindow.board.fieldStates.MastField;
+import com.battleships.gamewindow.board.fieldStates.SeaField;
+import com.battleships.gamewindow.services.RandomFleetPlacement;
+import com.battleships.models.board.Coordinate;
+import javafx.scene.layout.GridPane;
+
+import java.util.List;
+
+public class PlayerBoard extends Board {
+    private final RandomFleetPlacement randomFleetPlacement;
+
+    public PlayerBoard() {
+        super();
+        this.randomFleetPlacement = new RandomFleetPlacement();
+    }
+
+    public void changeAllFieldsToSea(BoardSize boardSize, GridPane gridPaneForBoard) {
+        for (int row = 1; row <= boardSize.rowsAmount(); row++) {
```

Comment:
why do you start with the first row?

Reply:
We count from 1, like here in column number
https://images.yourturnmyturn.com/rules/battleships/image004.gif

1 is more readeable for us